CONTRIBUTING.md

```diff
@@ -70,21 +70,21 @@ move forward. See editor Lee Byron talk about
 * **Enable new capabilities motivated by real use cases**
 
   Every change should intend on unlocking a real and reasonable use case. Real
-  examples are always more interesting than theoretical ones, and common
-  scenarios are more interesting than rare ones. RFCs should do more than offer
+  examples are always more compelling than theoretical ones, and common
+  scenarios are more compelling than rare ones. RFCs should do more than offer
```

Comment:
This is one of the more "meaningful" changes (if you will), because IMO theoretical use-cases are often _extremely_ interesting... they're just not compelling justifications for making a costly change. Real-world use cases, even if banal and boring, have much more weight, as the implementation/change cost is justified by usefulness.

Reply:
I agree with this change and its reasoning :+1: